scrapers/StateOfTexas.mjs

```diff
@@ -5,28 +5,35 @@ export default ({ database, DataScraper }) => {
 	return DataScraper(database, 'StateOfTexas', async () => {
 		const data = [];
 		const baseUrl = 'https://api.tnris.org/api/v1';
-		const id = '2679b514-bb7b-409f-97f3-ee3879f34448';
-		const collections = `collections/${id}`;
-		const resources = `resources?collection_id=${id}`;
-		const collectionsRequest = await fetch(`${baseUrl}/${collections}`);
-		const collection = await collectionsRequest.json();
-		const resourcesRequest = await fetch(`${baseUrl}/${resources}`);
-		const resource = await resourcesRequest.json();
+		const catalouge = await fetch(`https://api.tnris.org/api/v1/collections_catalog/`);
+		const catalougeData = await catalouge.json()
+		const datasets = catalougeData.results.map(dataset => {
+			return dataset.collection_id
```

Comment:
I would prefer to see the return replaced with the special fancy big arrow features. E.g. `dataset => dataset.collection_id`.